.github/workflows/vm-perf-to-prometheus.yml

```diff
@@ -21,7 +26,7 @@ jobs:
 
       - name: setup-env
         run: |
-          echo PUSH_VM_BENCHMARKS_TO_PROMETHEUS=1 >> .env
+          echo BENCHMARK_PROMETHEUS_PUSHGATEWAY_URL="http://vmagent.stage.matterlabs.corp/api/v1/import/prometheus/metrics/job/vm-benchmark" >> .env
```

Comment:
JFYI, this string was present in the source code previously. May makes sense to move it to repo env / secrets.

Reply:
Yes; please ping DevOps for that.